api/build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-params:${junitVersion}"
     testImplementation "org.junit.vintage:junit-vintage-engine:${junitVersion}"
     testImplementation "org.assertj:assertj-core:3.11.1"
-    testImplementation "com.h2database:h2:1.4.197"
+    testImplementation "io.zonky.test:embedded-database-spring-test:1.4.1"
```

Comment:
Why did you need to change this?

Reply:
Because I was getting an error initialising Flyway, and wanted to eliminate H2s a possible cause. Once I moved to Postgres I got a much more informative error back.